--- dinstaller/core/postscript.tmpl	2016-02-02 17:53:37.636847728 +0800
+++ dinstallern/core/postscript.tmpl	2016-02-02 18:00:34.588829139 +0800
@@ -99,6 +99,9 @@
 systemctl enable repairdev
 systemctl enable isoft-install-daemon
 systemctl enable sddm-plymouth
+systemctl disable umount-livecd
+rm -f /usr/lib/systemd/system/umount-livecd.service
+rm -f /usr/lib/systemd/scripts/umount-livecd.sh
 
 # ************************************************************
 # *** if disk is without flag boot, set one
